qiskit_nature/mappers/second_quantization/__init__.py

```diff
@@ -13,25 +13,9 @@
 """The Second Quantized Operator Qubit Mappers."""
 
 from .bravyi_kitaev_mapper import BravyiKitaevMapper
-from .bravyi_kitaev_super_fast_mapper import BravyiKitaevSuperFastMapper
-from .direct_mapper import DirectMapper
 from .fermionic_mapper import FermionicMapper
 from .jordan_wigner_mapper import JordanWignerMapper
 from .linear_mapper import LinearMapper
-from .logarithmic_mapper import LogarithmicMapper
 from .parity_mapper import ParityMapper
 from .qubit_mapper import QubitMapper
 from .spin_mapper import SpinMapper
-
-__all__ = [
```

Comment:
Why was this completely removed? This is normal to have in the init https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces

Also I would expect this file will have to contain sphinx documentation that is similar to what you see in other init files. That still needs doing for this and others I imagine.

Reply:
This is not handled consistently within Nature at the moment. I saw it was not there elsewhere and thus removed it.
I will make an effort to make this consistent instead.

I will also go through all `__init__` files and a `TODO(sphinx): documentation` comment which is easy to grep for when we get to the final preparation stages of this release. This is not something I am going to tackle in this PR.

Reply:
Taking care of the code is one thing. Given we often allow the user to import modules from its parent then not all init files need docs. We will have to look over the docs anyway and sort things out which yes would be a separate PR later once things are a bit more ready. As a first pass it needs at least the navigation structure in place - then that can be filled out. Anyway its for later....